### DISCUSSION TITLE
Font Color

### DISCUSSION THREAD

**Discussion by trace1729** [TheR1D/shell_gpt#265] [Q&A] [answered]
Is it possible to change output font color from purple to black/white?

**Comment by TheR1D** [comment, marked as answer]
> Is it possible to change output font color from purple to black/white?

Yes, it is possible by chaning `DEFAULT_COLOR` in runtime configuration file `~/.config/shell_gpt/.sgptrc` as it is described in [README.md](https://github.com/TheR1D/shell_gpt#runtime-configuration-file)

**Reply by ghost** [reply]
is it possible to make it not bold please?
